partner_analytic/models/res_partner.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    property_analytic_account_id = fields.Many2one(
+        comodel_name="account.analytic.account",
+        company_dependent=True,
+        string="Customer Analytic Account",
+        help="Default Analytic Account used for Invoices / Credit Notes.",
+    )
+
+    property_supplier_analytic_account_id = fields.Many2one(
+        comodel_name="account.analytic.account",
+        company_dependent=True,
+        string="Vendor Analytic Account",
+        help="Default Analytic Account used for Vendor Bills / Vendor Refund.",
+    )
```

Comment:
to align them with other account fields in partner we should make them commercial fields to only be managed at the parents

Reply:
done!